docs/case-studies/jenkins-cicd.md

```diff
@@ -2,19 +2,19 @@
 title:  Jenkins + KubeVela for CI/CD
 ---
 
-This section will introduce how to combine KubeVela and Jenkins to do CI (continuous integration) and CD (continuous delivery). The following example is based on the publish process of a simple HTTP server. The code can be found at [here](https://github.com/Somefive/KubeVela-demo-CICD-app).
+This section will introduce how to combine KubeVela with Jenkins to do CI (continuous integration) and CD (continuous delivery). The following example is based on the release process of a simple HTTP server. The code can be found at [here](https://github.com/Somefive/KubeVela-demo-CICD-app).
 
 ## Preparation
 
 Before combining KubeVela and Jenkins, you need to ensure the following environments have already been set up.
 
 1. Deploy Jenkins service with Docker support, including related plugins and credentials which will be used to access image repository.
-2. A git repository with Webhook enabled. Ensure commits to the target branch will trigger the running of Jenkins pipeline.
-3. Get Kubernetes for deployment. Install KubeVela and enable its apiserver. Ensure the KubeVela apiserver can be accessed from external.
+2. A git repository with Webhook enabled. Ensure commits to the target branch will trigger the running of the Jenkins pipeline.
+3. Get Kubernetes for deployment. Install KubeVela and enable its apiserver. Ensure the KubeVela apiserver can be accessed from external environment.
```

Comment:
How to make the APIServer exposed outside?

Reply:
https://github.com/yangsoon/kubevela/blob/deb6d79f61e5f616a6635eeef260d226f901b3e9/charts/vela-core/values.yaml#L111
It is exposed by ClusterIP now. Users could configure Ingress to use it. @yangsoon is writing docs for more details about install apiserver, but not here, at the advance installation section.